pom.xml

```diff
@@ -435,5 +436,30 @@ SOFTWARE.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <!-- Deep Testing: https://www.yegor256.com/2023/08/22/fast-vs-deep-testing.html -->
+      <id>deep</id>
```

Comment:
@h1alexbel It's a good thing actually! However I suggest to keep in this PR only the changes relevant to the #101 issue. We can add this `deep` profile in a separate PR. What do you think?

Reply:
Agree